Modules/stubs/CreateMigration.stub

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class {{ class }} extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('{{ table }}', function (Blueprint $table) {
```

Comment:
I could not find the table variable, should it be `{{ table }}s` ? Plural according to laravel's convention?

Reply:
> Thanks for the PR - Really appreciate it. The code and the documentation were mindblowing :)
> 
> But I want to keep Hydra totally lean and less opinionated so that user can have their own freedom to pick any module system or write their own if they really need one.
> 
> Let's see what others say about this.

Is okay as you want to keep Hydra dry, but in all my recent projects I hardly use Laravel the way it is, I like Modular Development, so I crafted out this that helps me to work faster, I wanted to open source it so other developers can use it as well but I haven't gotten any better time/chance to do that. When I saw hydra on my daily dev feed, I opened and scanned through the codes, I found out its still an infant project and might need more features. So I duplicates some of the implementations of what I have been working on here, though I made some changes to suit with the project laravel version.

Am the kind of developer that likes customizing whatever am using to my taste and when the tool/framework that I uses won't let me to do that am off, at the same time I like to whatever am developing to be of the same nature, you can decide to use or not. So we can make this work as hidden feature that the user can use custom artisan command to publish and use.

I still have some features that I will like to add, everything will be pluggable, so that a developer can plug in any feature or plug out what he doesn't need to make it simpler, that is the plan

I still have (plugins)
1. Image Optimizer with Spatie image optimizer
2. Media Manager that attaches media to a model, this works with Image optimizer (optional), Media can be saved in any cloud storage service, currently supports only cloudinary
3. Role management made easy (with spatie)
4. More laravel request Rules